### ISSUE TITLE
Remove inboxes and ActivityStreams JSON

### ISSUE THREAD

**Issue by acka47** [skohub-io/skohub-vocabs#171]
SkoHub PubSub (as well as SkoHub Editor) hasn't been used in production settings yet. Because of this and because of missing resources we'd need for maintenance, we decided to switch off both modules on skohub.io for an indefinite period. 

Against this background, we should better cleanly remove all PubSub related functions from SkoHub Vocabs, which are at least these:

- [x] `jsonas`, `inbox` and `followers` stuff in `gatsby-node.js`, `src/queries.js`, `/.env.example`, `src/common.js`, `/src/types.js`
- [x] inbox link in `/src/components/Concept.js`
- [x] inbox mention in `README.md`

**Comment by acka47** [comment]
Here is my functional review of the changes to be introduced with #199:

- When triggering a build for the first time, I got an error message, see https://test.skohub.io/build/?id=7193fc66-3a32-42f0-8880-cf8a8197c549. However, this must not be related to the changes.
- When I triggered the build again, everything ran as expected: https://test.skohub.io/build/?id=b39ffa40-f830-423e-9dc0-5f15a1f1f12a
- Furthermore, I had now problems when setting up a development server locally.
- I could not find any remnants of ActivityStream data, the `.jsonas` files are completely removed, `followers` and `inbox` are gone from the `.json` and the inbox link isn't shown anymore in the HTML. (Also, the inbo mention from the README is gone.)

Over all, +1 fom my side.
